tools/racon/main.nf

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env nextflow
+
+// Specify DSL2
+nextflow.enable.dsl = 2
+
+// Process definition
+process racon {
+    publishDir "${params.outdir}/${opts.publish_dir}",
+        mode: "copy",
+        overwrite: true,
+        saveAs: { filename ->
+                      if (opts.publish_results == "none") null
+                      else filename }
+
+    container "luslab/nf-modules-racon:latest"
+
+    input:
+        val opts
+        tuple val(meta), path(reads), path(overlap_paf), path(assembly_fasta)
```

Comment:
this is generally not the correct structure - you want to pass these files as separate channels unless they are directly linked to the reads file?